cmd/adhoc/cmd/root.go

```diff
@@ -24,6 +28,30 @@ func init() {
 	)
 }
 
+func readFile() {
+	// Setting up file formatting
+	viper.SetConfigFile("syncron.yaml")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+	// Reading from file
+	err := viper.ReadInConfig()
+	if err != nil {
+		logrus.Error("Config file not found.\n")
```

Comment:
Think of logs as something complementary that you can remove and have the app running as if they were on. For this type of comment, which the user needs to know, you probably want to print into stdout instead.